CONTRIBUTING.md

```diff
@@ -24,11 +24,11 @@ You need to be enabled for doing this.
 * `master` should be all green. If not, make it green first.
 * git checkout master
 * git pull --tags
-* Run `grunt version:<versionType>` for new version.
+* Run `npm run version -- --type="<versionType>"` for new version.
```

Comment:
I was thinking that for the sake of simplicity, we could simplify this by having `npm run patch`, `npm run minor` and `npm run major`? What do you think

Reply:
I agree, the current script is unnecessarily verbose.
I'm happy to update this so we can keep it within a single PR.